tests/test_interface.py

```diff
@@ -85,77 +118,181 @@ def test_InterfaceChangeMtu(self, dvs):
                 for fv in fvs:
                     if fv[0] == "SAI_ROUTER_INTERFACE_ATTR_TYPE":
                         assert fv[1] == "SAI_ROUTER_INTERFACE_TYPE_PORT"
+                    # assert the new value set to the router interface
                     if fv[0] == "SAI_ROUTER_INTERFACE_ATTR_MTU":
                         assert fv[1] == "8888"
 
-    def test_InterfaceRemoveIpv4Address(self, dvs):
-        pdb = swsscommon.DBConnector(0, dvs.redis_sock, 0)
-        adb = swsscommon.DBConnector(1, dvs.redis_sock, 0)
-        cdb = swsscommon.DBConnector(4, dvs.redis_sock, 0)
-
         # remove IP from interface
-        tbl = swsscommon.Table(cdb, "INTERFACE")
-        tbl._del("Ethernet8|10.0.0.4/31")
+        self.remove_ip_address("Ethernet16", "20.0.0.8/29")
+
+class TestLagRouterInterfaceIpv4(object):
+    def setup_db(self, dvs):
+        self.pdb = swsscommon.DBConnector(0, dvs.redis_sock, 0)
+        self.adb = swsscommon.DBConnector(1, dvs.redis_sock, 0)
+        self.cdb = swsscommon.DBConnector(4, dvs.redis_sock, 0)
+
+    # TODO: below three functions will be replaced with configuration
+    # database updates after the future changes of the lagmgrd
+    def create_port_channel(self, dvs, interface):
+        dvs.runcmd("teamd -d -t" + interface)
+        time.sleep(1)
+
+    def remove_port_channel(self, dvs, interface):
+        dvs.runcmd("teamd -k -t" + interface)
+        time.sleep(1)
+
+    def add_port_channel_members(self, dvs, interface, members):
```

Comment:
Can we also have a test case to remove_port_channel_members?

Reply:
will add this along with port channel membership configuration changes